fvm/src/syscalls/error.rs

```diff
@@ -44,6 +46,11 @@ impl Abort {
             ExecutionError::Syscall(e) => Abort::Fatal(anyhow!("unexpected syscall error: {}", e)),
         }
     }
+
+    /// Convert a generalized exit "jump" into an "abort".
+    pub fn from_exit(code: ExitCode, msg: String, blk: BlockId) -> Self {
+        Abort::Exit(code, msg, blk)
+    }
```

Comment:
This isn't really necessary. I'd just call `Abort::Exit(code, msg, blk)` directly.

Reply:
ok, will drop.

Reply:
removed.